build.ps1

```diff
@@ -14,7 +14,7 @@ param(
     [string] $CefBinaryDir = "../cefsource/chromium/src/cef/binary_distrib/",
 
     [Parameter(Position = 3)]
-    $CefVersion = "3.2883.1552.g88ff29a"
+    $CefVersion = "3.2924.1575.g97389a9"
```

Comment:
Upgrading of versions would ideally be in a separate `PR` as it's unrelated to adding `VS2017` support.